contents/data/nosql/NoSQL.md

```diff
@@ -0,0 +1,436 @@
+<frontmatter>
+  title: Introduction to NoSQL
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to NoSQL
+
+**Author(s): [Ang Ze Yu](https://github.com/ang-zeyu)**
+
+**Reviewer(s): [Neil Brian](https://github.com/nbriannl), [James Pang](https://github.com/jamessspanggg), [Daryl Tan](https://github.com/openorclose), [Yash Chowdhary](https://github.com/yash-chowdhary)**
+
+<box id="article-toc">
+
+* [Introduction to NoSQL](#introduction-to-nosql)
+  * [What is NoSQL‎](#what-is-nosql)
+  * [How does NoSQL work?‎](#how-does-nosql-work)
+    * [Basic structure of document-based databases‎](#basic-structure-of-document-based-databases)
+    * [Basic CRUD operation examples‎](#basic-crud-operation-examples)
+    * [Schemas in NoSQL](#schemas-in-nosql)
+    * [Relation storage in NoSQL](#relation-storage-in-nosql)
+    * [Horizontal Scaling](#horizontal-scaling)
+  * [Why NoSQL](#why-nosql)
+    * [1. Highly suited for agile development](#1-highly-suited-for-agile-development)
+    * [2. Horizontal scalability](#2-horizontal-scalability)
+    * [3. Widespread adoption](#3-widespread-adoption)
+  * [Caveats of NoSQL‎](#caveats-of-nosql)
+    * [Lack of standardisation](#1-lack-of-standardisation)
+    * [Not suited for complex relational queries‎](#2-not-suited-for-complex-relational-queries)  
+  * [How to get started with NoSQL?‎](#how-to-get-started-with-nosql)
+
+{.px-3}
+
+</box>
+
+---
+
+## What is NoSQL?
+
+**Non-Structured Query Language** (**NoSQL**) is a wide set of implementations of query technologies used to retrieve and store data in a non-tabular format.
+
+Some common implementations of such databases are key-value stores, document stores (which are an extension of key-value stores),
+and though less common - graph databases (which can also be considered an extension of document stores).
+
```

Comment:
If the reader can understand this section, most likely she is knowledgeable enough in the database domain to know what NoSQL is. Tech terms density is very high too. Need to make the introduction gentler and gradual. You can also declare your assumptions about prior knowledge at the start.
One option is to declare the SQL chapter is a prerequisite, and write this chapter as if the reader is a beginner but has read the SQL chapter.